hack/local-development/start-apiserver

```diff
@@ -58,7 +58,6 @@ apiserver_flags="
   --tls-private-key-file $TLS_KEY_FILE \
   --feature-gates SeedChange=true \
   --feature-gates AdminKubeconfigRequest=true \
-  --feature-gates UseDNSRecords=true \
```

Comment:
Deletion of this line is not required because with the beta version user has the access to enable or disable the feature gate.

Reply:
User can always add this line, right?
Also, I couldn't find the same for other Beta components.
And since it's for the api-server to start with UseDNSRecords enabled, and now it's already enabled, so felt it isn't needed.

Reply:
/second-opinion